templates/report.html

```diff
@@ -0,0 +1,142 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Report</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css">
+    <link rel="stylesheet" type="text/css" href={{ css_url }} />
+</head>
+    <body>
+        <h1>Video Metadata Report</h1>
+        <div class="card container">
+            {% for video in stream_info.video_stream %}
+            <div class="row">
+                <div class="col-3">
+                    <img class="videoImg" src={{ stream_info.video_img_url }} alt="Avatar">
```

Comment:
alt text should not be "Avatar".

Reply:
I changed it to a reasonable attribute name.